packages/bpk-component-calendar/src/BpkCalendarNav.js

```diff
@@ -81,17 +84,20 @@ const BpkCalendarNav = props => {
         >
           <ArrowLeftIcon className={getClassName('bpk-calendar-nav__icon')} />
           <span className={getClassName('bpk-calendar-nav__text--hidden')}>
-            {formatMonth(addMonths(baseMonth, -1))}
+            {previousMonthLabel || formatMonth(prevMonth)}
           </span>
         </button>
       </div>
       <div className={getClassName('bpk-calendar-nav__month')}>
         <label
           htmlFor={`${id}_select`}
           className={getClassName('bpk-calendar-nav__text--hidden')}
+          aria-hidden="true"
         >
           {changeMonthLabel}
         </label>
+
+        <BpkAriaLive aria-hidden="true">{formatMonth(baseMonth)}</BpkAriaLive>
```

Comment:
`aria-hidden="true"` In this case, we don't want the live region to be read out unless it's changed, otherwise this results in duplication of what is read out.